src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,126 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './TodoList.scss';
+import classNames from 'classnames';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.Component {
+  state = {
+    title: '',
+    status: '',
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
+
+  render() {
+    const { todos, selectUser, userId, changeStatus } = this.props;
+    const { title, status } = this.state;
+    let filterTodos = todos;
+
+    if (title) {
+      filterTodos = todos.filter(
+        todo => todo.title && (
+          todo.title.toLocaleLowerCase().includes(title.toLocaleLowerCase())),
+      );
+    }
+
+    switch (status) {
+      case 'Active':
+        filterTodos = filterTodos.filter(
+          todo => todo.completed !== null && !todo.completed,
+        );
+        break;
+      case 'Completed':
+        filterTodos = filterTodos.filter(
```

Comment:
For the future - it would be better to make all operations (filtering) in the upper level and pass ALREADY prepared todos to the 'TodoList' component